ts/nni_manager/training_service/reusable/environments/kubernetes/kubeflowEnvironmentService.ts

```diff
@@ -26,6 +26,7 @@ export class KubeflowEnvironmentService extends KubernetesEnvironmentService {
         // Create kubernetesCRDClient
         this.kubernetesCRDClient = KubeflowOperatorClientFactory.createClient(
             this.config.operator, this.config.apiVersion);
+        this.kubernetesCRDClient.setNamespace = this.config.namespace ? this.config.namespace : "default";
```

Comment:
The JS way: `this.config.namespace ?? 'default'`

Reply:
got it